named_entity_extractor.py

```diff
@@ -0,0 +1,112 @@
+""" Wrapper for all GCP Language API code.
+
+You can read more about it here: https://cloud.google.com/natural-language/docs/basics
+You will have to export GOOGLE_APPLICATION_CREDENTIALS before running this file.
+export GOOGLE_APPLICATION_CREDENTIALS=[PATH TO credentials.json]
```

Comment:
does the google cloud lib read this env var?

Reply:
It used to before I added the json parser.